okapi-core/src/main/java/org/folio/okapi/MainVerticle.java

```diff
@@ -214,16 +215,12 @@ public void start(Promise<Void> promise) {
 
   private Future<Void> startModmanager() {
     logger.info("startModmanager");
```

Comment:
Consider to use uppercase M for manager part

Reply:
ok. Renaming the function.